### PR TITLE
Several Profile Improvements

### PR DESCRIPTION
**Pull request by laurencelundblade**
Address freshness
Address CBOR preferred serialization
Use CDDL in profile documents

### CODE REVIEW

**Comment by laurencelundblade** [comment]
The nonces claim should reference RATS architecture for freshness too.

File: draft-ietf-rats-eat.md [hunk 1]
```diff
@@ -1182,6 +1196,12 @@ The profile document should go into the full detail.
 Similar to, or perhaps the same as Verification Key Identification, the profile may wish to specify how Endorsements are to be identified.
 However note that Endorsement Identification is optional, where as key identification is not.
 
+### Freshness
+
+Just about every use case will require some means of knowing the EAT is recent enough and not a replay of an old token.
+The profile should describe how freshness is achieved.
+The section on Freshness in {{RATS-Architecture}} describes several solutions to achieve this.
```
Comment (thomas-fossati) [inline draft-ietf-rats-eat.md:1203]: Maybe we could make it clearer here that one is not bound to the freshness models examined in {{RATS-Architecture}}.  E.g., a protocol embedding EAT could use a time notion based on some kind of event stored in Merkle tree, or anything else really as long as it fits the bill.

**Review by thomas-fossati — COMMENTED** [review]
lgmt, I've left a couple of minor comments

File: draft-ietf-rats-eat.md [hunk 2]
```diff
@@ -1146,6 +1154,12 @@ A good default is to allow only definite-length strings.
 As with map and array encoding, allowing indefinite-length strings can be beneficial for some smaller implementations.
 
 
+### CBOR Preferred Serialization
+
+The profile should indicate whether encoders must use preferred serialization.
+The profile should indicate whether decoders must accept non-preferred serialization.
```
[the remark this answers is not in the document]
Comment (laurencelundblade) [reply draft-ietf-rats-eat.md:1160]: All the things in the profile are described as "should" right now.

If we are going to start using "must" then we are describing a formal protocol for creating profile documents to the degree that some profiles comply with EAT and some don't. That seems like an overreach and a tall order.

My leaning is to leave it as "should", but we can have a larger discussion about more formalism here. Maybe do this by filing a separate issue for it.

[the remark this answers is not in the document]
Comment (thomas-fossati) [reply draft-ietf-rats-eat.md:1160]: OK!  I raised #100 to discuss this.

**Pull request merged**